cheshire_cat_api/models/body_upload_url.py

```diff
@@ -32,7 +32,7 @@ class BodyUploadUrl(BaseModel):
 
     class Config:
         """Pydantic configuration"""
-        allow_population_by_field_name = True
+        populate_by_name = True
```

Comment:
most of this code is auto-generated with openapi generator, I would leave this and the other properties names unchanged to avoid conflicts when updating the endpoints